src/inspector_agent.cc

```diff
@@ -429,6 +430,33 @@ class ChannelImpl final : public v8_inspector::V8Inspector::Channel {
   std::unique_ptr<v8_inspector::V8InspectorSession> session_;
 };
 
+class RepeatingInspectorTimer {
+ public:
+  RepeatingInspectorTimer(uv_loop_t* loop, double interval,
+                 v8_inspector::V8InspectorClient::TimerCallback callback,
+                 void* data) : timer_(uv_timer_t()), callback_(callback),
+                               data_(data) {
+    uv_timer_init(loop, &timer_);
+    int64_t interval_ms = 1000 * interval;
```

Comment:
This is just a question, not a request to change anything: Does V8 have any documention saying that this `interval` is supposed to be given in seconds? Where would one find that?